### PR TITLE
Altcoin sync(LL-7086)

### PR DESCRIPTION
**Pull request by hzheng-ledger**
https://ledgerhq.atlassian.net/browse/LL-7086

### CODE REVIEW

File: src/account.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ export interface Account {
   params: {
     path: string;
     index: number;
+    currency?: string;
```
Comment (machard) [inline src/account.ts:7]: why optional ?

Comment (hzheng-ledger) [reply src/account.ts]: yes, you are right.
I've remove optional.

File: src/crypto/types.ts [hunk 2]
```diff
@@ -13,3 +13,23 @@ export interface ICrypto {
 
   toOutputScript(address: string): Buffer;
 }
+
+export type Currency =
```
Comment (machard) [inline src/crypto/types.ts:17]: I think you meant to use this type for the currency property but instead you used string ?

Comment (hzheng-ledger) [reply src/crypto/types.ts:17]: Fixed. The code can be reviewed again. Thanks

**Review by machard — APPROVED** [review]

**Pull request merged**